components/camel-as2/camel-as2-api/src/main/java/org/apache/camel/component/as2/api/AS2ClientConnection.java

```diff
@@ -89,15 +110,61 @@ public String getClientFqdn() {
 
     public HttpResponse send(HttpRequest request, HttpCoreContext httpContext) throws HttpException, IOException {
 
+        // Connection pooling
+        ConnectionRequest connectionRequest = connectionManager.requestConnection(httpRoute, null);
+        HttpClientConnection httpClientConnection = null;
+        try {
+            httpClientConnection = connectionRequest.get(10, TimeUnit.SECONDS);
+        }  catch (InterruptedException | ExecutionException e) {
+            throw new HttpException(e.getMessage(), e.getCause());
+        }
+
+        connectionManager.connect(httpClientConnection, httpRoute, 5000, httpContext);
```

Comment:
is the number here configurable? the same way as @dmvolod mentioned in earlier configurations